src/xdocs/config_coding.xml

```diff
@@ -4303,6 +4320,32 @@ for(int i=0; i&lt;10; i++) {
   &lt;property name=&quot;max&quot; value=&quot;3&quot;/&gt;
 &lt;/module&gt;
         </source>
+        <p>Valid Code Example:</p>
+        <source>
+if (x &gt; 2) {
+   if (x &gt; 3) {
+      if (x &gt; 4) {
+         if (x &gt; 5) {}
+         else {} // OK
```

Comment:
Instead of writing `//OK` after `else` block, write it after `if` block.

Reply:
done.